.gitignore

```diff
@@ -1 +1,5 @@
-.drone.sec.yaml
+# Folders to ignore
+.vscode/
+
+# Files to ignore
+.drone.sec.yaml
```

Comment:
Please add a new line.

Reply:
Great thanks a lot for pointing that out!